bodhi/server/models.py

```diff
@@ -2346,8 +2361,8 @@ def check_requirements(self, session, settings):
                     return False, "Required task %s returned %s" % (
                         latest['testcase']['name'], latest['outcome'])
 
-        if config.get('ci.required') and not self.ci_passed:
-            return False, "CI did not pass on this update"
+        if config.get('greenwave_enabled') and not self.greenwave_policies_satisfied:
+            return False, "Greenwave policies are not satisfied on this update"
```

Comment:
Let's s/Greenwave/Testing/, so that users not familiar with Greenwave will understand what this means.